### PR TITLE
Add possibility to enable and test PMU feature

### PR DESCRIPTION
**Pull request by LKomaryanskiy**

### CODE REVIEW

File: meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-domd.dts [hunk 1]
```diff
@@ -45,6 +45,14 @@
 	 * CPU nodes to be present get broken. This is why we have to explicitly
 	 * remove the following.
 	 */
+	/*
+	 * Remove interrupts-extended node from guest domain
+	 * as it can not be shared by Xen.
+	 */
+	pmu_a76 {
+		compatible = "arm,cortex-a55-pmu";
```
Comment (lorc) [inline meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-domd.dts:53]: Do you really need to override `compatible`?

File: prod-devel-rcar-s4.yaml [hunk 2]
```diff
@@ -79,6 +79,7 @@ common_data:
 
     # Generate ext4 image files
     - [IMAGE_FSTYPES:append, " ext4"]
+    - [IMAGE_INSTALL:append, " perf"]
```
Comment (lorc) [inline prod-devel-rcar-s4.yaml:82]: IIRC, we are doing this by image recipe, not via YAML file.

File: doc/virtualization.md [hunk 3]
```diff
@@ -201,3 +201,11 @@ DHCP. User can provide own IP address by editing
...
 by un-commenting corresponding line in `/etc/xen/domu.cfg` file in
 Dom0.
+
+## PMU
+Note, this feature is not security supported:
+see https://xenbits.xen.org/xsa/advisory-163.html.
+It's not recommended to use it for production builds
+but it may be  used for development or debugging.
+PMU is disabled by default. It can be enabled by by un-commenting corresponding
+lines in `/etc/xen/domu.cfg` or `/etc/xen/domd.cfg` file in Dom0.
```
Comment (lorc) [inline doc/virtualization.md:211]: Could you please add a couple more paragraphs?
1. Describe why do we need pin vCPUs
2. Describe why it is not securely supported

**Review by lorc — CHANGES_REQUESTED** [review]

File: meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-domd.dts (hunk 1 above)
Comment (firscity) [reply meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-domd.dts:53]: if so, I think it is better to completely remove existing node (invalid naming) and provide new with correct name and proper `compatible`.

Comment (LKomaryanskiy) [reply meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-domd.dts]: No, we do not need to override `compatible`, it's a copy-paste issue. I removed this line.

File: prod-devel-rcar-s4.yaml (hunk 2 above)
Comment (LKomaryanskiy) [reply prod-devel-rcar-s4.yaml:82]: As I see, we still add the required packages by yaml file:
https://github.com/xen-troops/meta-xt-prod-devel-rcar-gen4/blob/master/prod-devel-rcar-s4.yaml#L150
Or should it should be moved to some other file?

File: doc/virtualization.md (hunk 3 above)
Comment (LKomaryanskiy) [reply doc/virtualization.md:213]: Thank you - added descriptions:)

**Comment by LKomaryanskiy** [comment]
@lorc, @firscity  thank you for your review comments. I released a new version of PR that includes the following changes:
1. Removed overriding `compatible` field;
2. Added more details about enabling PMU feature.

Please, take a look at the new version.

**Review by firscity — APPROVED** [review]
`Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>`

File: prod-devel-rcar-s4.yaml (hunk 2 above)
Comment (lorc) [reply prod-devel-rcar-s4.yaml:82]: Ah, okay. Let it be here.

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by LKomaryanskiy** [comment]
@lorc @firscity thank you for your reviews. Added acked-by tags.

**Pull request merged**